### DISCUSSION TITLE
催更，快更新一下吧。感觉大家都用不了了。

### DISCUSSION THREAD

**Discussion by colatea** [huan/docker-wechat#79] [General]
好多都是exit 0的问题。

**Comment by Echoscorner** [comment]
+ '[' -n true ']'
+ wine reg query 'HKEY_CURRENT_USER\Software\Tencent\WeChat'
reg��ϵͳ�޷��ҵ�ָ����ע��������ֵ
+ echo 'Register for Wechat not found ?'
+ exec wine 'C:\Program Files\Tencent\WeChat\WeChat.exe'
Register for Wechat not found ?

📦 DoChat Exited with code [0]


对，上面是debug模式启动，同遇到  exit code0

**Comment by huan** [comment]
Thanks for pushing the project moving forward!

I would love to publish a new version if anyone can send a PR to fix those problems, I'm looking forward to seeing that!

**Comment by colatea** [comment]
2.7.1.85: Pulling from zixia/wechat
Digest: sha256:e6e9d21c7cd1dfae0484e697f12f5f3c401de2f02e771d061868740e0d26549d
Status: Image is up to date for zixia/wechat:2.7.1.85
docker.io/zixia/wechat:2.7.1.85

🚀 Pulling the docker image done.
🚀 Starting DoChat /dɑɑˈtʃæt/ ...


📦 DoChat Exited with code [0]

-----------------------------------------------
更新了，还是无法使用。

██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: 10S4S02L00 ThinkCentre M920x-N000 
████████            ████████   Kernel: 5.8.11-1-MANJARO 
████████  ████████  ████████   Uptime: 1 hour, 47 mins 
████████  ████████  ████████   Packages: 1267 (pacman) 
████████  ████████  ████████   Shell: zsh 5.8 
████████  ████████  ████████   Resolution: 1920x1200 
████████  ████████  ████████   DE: GNOME 
████████  ████████  ████████   WM: Mutter 
████████  ████████  ████████   WM Theme: Matcha-dark-sea 
████████  ████████  ████████   Theme: Matcha-sea [GTK2/3] 
████████  ████████  ████████   Icons: Papirus-Dark-Maia [GTK2/3] 
                               Terminal: gnome-terminal 
                               CPU: Intel i5-9500 (6) @ 4.400GHz 
                               GPU: Intel UHD Graphics 630 
                               Memory: 2943MiB / 15848MiB

**Comment by wlnirvana** [comment]
Not sure if this would work for you, but removing `~/DoChat` before starting wechat solved my issue.

**Comment by huan** [comment]
Link to #131
